test/functional/ui/wildmode_spec.lua

```diff
@@ -55,13 +55,15 @@ describe("'wildmenu'", function()
   it('is preserved during :terminal activity', function()
     -- Because this test verifies a _lack_ of activity after screen:sleep(), we
     -- must wait the full timeout. So make it reasonable.
+    -- XXX: Unreliable screen output in Appveyor
+    if helpers.pending_win32(pending) then return end
```

Comment:
Let's keep this line in the same place as before, to avoid churn in the git logs where possible.